components/Feed/FeedList/FeedList.tsx

```
@@ -1,22 +1,27 @@
 import relativeTime from "dayjs/plugin/relativeTime";
 import { cleanupTextContent } from "@/utils/htmlUtils";
 dayjs.extend(relativeTime);
 
 interface FeedListProps {
-  items: FeedItem[];
-  isLoading: boolean;
-  selectedItem?: FeedItem | null;
-  onItemSelect: (item: FeedItem, scrollTop: number) => void;
-  savedScrollPosition?: number;
+  // No props needed - uses context
 }
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)